scripts/main.js

```diff
@@ -0,0 +1,57 @@
+let books;
+// eslint-disable-next-line
+if (localStorage.getItem('books') === null) {
+  books = [];
+} else {
+  // eslint-disable-next-line
+  books = JSON.parse(localStorage.getItem('books'))
+}
+
+document.querySelector('form').addEventListener('submit', () => {
+  const bookTitle = document.querySelector('.title').value;
+  const bookAuthor = document.querySelector('.author').value;
+
+  const book = [{ bookTitle, bookAuthor }];
+  books.push(book);
+  // eslint-disable-next-line
+  localStorage.setItem('books', JSON.stringify(books))
+});
+
+document.addEventListener('DOMContentLoaded', () => {
+  books.forEach((book) => {
+    const bookList = document.querySelector('.books-card');
+
+    bookList.innerHTML += `
+    <div class="books-card-flex">
+    <h3 class="book-title">${book[0].bookTitle}</h3>
+    <p class="book-about">${book[0].bookAuthor}</p>
+    <button type="button" class="button-remove">Remove</button>
+    <br><hr>
+    </div>
+    `;
+  });
+});
+
+document.addEventListener('DOMContentLoaded', () => {
+  const buttonRemove = document.querySelectorAll('.button-remove');
+
+  buttonRemove.forEach((button) => {
+    button.addEventListener('click', (e) => {
+      const bookAuthor = e.target.previousElementSibling.innerText;
+      const bookTitle = e.target.previousElementSibling.previousElementSibling.innerText;
+
+      // eslint-disable-next-line no-plusplus
+      for (let i = 0; i < books.length; i++) {
+        if (books[i][0].bookTitle === bookTitle && books[i][0].bookAuthor === bookAuthor) {
+          books.splice(i, 1);
```

Comment:
[OPTIONAL] Consider using methods that don't mutate the array.

<img width="960" alt="Screen Shot 2022-08-02 at 7 53 26" src="https://user-images.githubusercontent.com/82426682/182379428-b488af6f-d4ce-4200-bc0e-af7ccde79bc2.png">